scripts/istat/geos/test.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+
+def testColnames(csv_path, tmcf_path):
```

Comment:
Ooh.. I like this... if you see a way to make this a general test for everything in this repo, let me know!

Reply:
If we follow the naming rule that the cleaned CSV file and template mcf file has the same name but different extensions. We can apply this test to all the tmcf files found in the repo.